test/gulpfile.js/index.js

```diff
@@ -0,0 +1,45 @@
+/* eslint promise/prefer-await-to-then: 0 */
```

Comment:
found it a little odd to have the dir named `gulpfile.js`, but i think it's ok. slightly prefer `gulpfile-js` i think.

Reply:
This was `gulp`s suggestion. But it turns out it only works in the root of the project (you can replace `gulpfile.js` with a directory `gulpfile.js` and it will use the index in there autoatically).

Reply:
oh interesting! if i were gulp i would look for a `gulpfile` dir since you can always `require(dir)` or `require(file_without_js_extension)` in node anyway